tock/tock/templates/_navigation.html

```diff
@@ -10,11 +10,24 @@
           <span>Projects</span>
         </a>
       </li>
-      <li>
-        <a href="{% url 'reports:ListReports' %}" class="usa-nav-link">
-          <span>Reports</span>
-        </a>
-      </li>
+      {% if user.is_staff %}
```

Comment:
This should be `request.user.is_staff`

Reply:
Huh. I thought we passed user into context, but it appears you're right!